src/docs/content/observability/observability-scenario/pre-requisites/setup.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 1
+tags:
+  - mushop setup
+---
+
+## Pre-requisites
+
+### Install Ingress Controller
+
+```bash
+kubectl create namespace mushop
+```
+```text
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/mandatory.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/provider/cloud-generic.yaml
+```
+
+### Create a Secret for OCI Monitoring
+
+```bash
+kubectl create secret generic monitoring-secret \
+--from-literal=compartment_id="<your compartment_id>" \
+--from-literal=tenant_id="<your tenancy_id>" \
+--from-literal=user_id="<your user_id>" \
+--from-literal=fingerprint="<your fingerprint>" \
+--from-literal=passphrase="<your passphrase>" \
+--from-literal=monitoring_endpoint="<your monitoring endpoint>" \
+--from-literal=region="<your region>" \
+--from-file=apikey=<your api_key.pem>
+```
+
+## Deploy
+
+```
+cd deploy/complete/kubernetes
+```
+
+```text
+kubectl apply -f mushop.yaml
```

Comment:
Why suggesting to use regular manifest and not helm charts?

Reply:
To use enhanced mock mode which captures communication between different services and logging excluding catalogue. we needed this to implement the payment failure scenario.
the basic mock is only limited to API service.